scoring/management/commands/update_coherence_tournament_leaderboard.py

```
@@ -0,0 +1,806 @@
+    logger.info("Mean: %.2f", skills_array.mean())
+    logger.info("Std: %.2f", skills_array.std())
+    logger.info("Skewness: %.4f", stats.skew(skills_array))
+    logger.info("Kurtosis: %.4f", stats.kurtosis(skills_array))
+
+
+class Command(BaseCommand):
+    help = """
+    Update the global bots leaderboard
+    """
+
+    def handle(self, *args, **options) -> None:
```

Ruff (0.14.14)

[warning] Unused method argument: `args`

(ARG002)